tests/bluetooth/tester/src/btp/btp_csip.h

```diff
@@ -22,3 +22,35 @@ struct btp_csip_discover_cmd {
 struct btp_csip_start_ordered_access_cmd {
 	uint8_t flags;
 } __packed;
+
+#define BTP_CSIP_SET_COORDINATOR_LOCK		0x04
+struct btp_csip_set_coordinator_lock_cmd {
+	uint8_t count;
+} __packed;
+
+#define BTP_CSIP_SET_COORDINATOR_RELEASE	0x05
+struct btp_csip_set_coordinator_release_cmd {
+	uint8_t count;
+} __packed;
```

Comment:
These does not need any parameter to identity a specific set?

Reply:
There was no need to set lock/release lock on a specific set. Maybe I'll add comment that this might need update in the future

Reply:
That makes sense. I don't expect any tests to work on multiple sets